scripts/verify-structure.py

```
@@ -496,12 +644,61 @@ def check_custom_component_paths(plugin_dir: Path, plugin_data: dict) -> list[st
+        'repository', 'license', 'keywords'
+    ]
+
+    for field in comparable_fields:
+        market_value = marketplace_entry.get(field)
+        plugin_value = plugin_json_data.get(field)
+
+        # Both exist and differ
+        if market_value and plugin_value and market_value != plugin_value:
+            warnings.append(
+                f"{plugin_name}: Conflict in '{field}' - "
+                f"marketplace: {repr(market_value)}, "
```

Ruff (0.14.4)

Use explicit conversion flag

Replace with conversion flag

(RUF010)